lambda/service/handler/handler.go

```diff
@@ -0,0 +1,56 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"log"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/pennsieve/integration-service/service/clients"
+	"github.com/pennsieve/integration-service/service/models"
+	"github.com/pennsieve/integration-service/service/store"
+	"github.com/pennsieve/integration-service/service/trigger"
+)
+
+func IntegrationServiceHandler(ctx context.Context, request events.APIGatewayV2HTTPRequest) (events.APIGatewayV2HTTPResponse, error) {
+	body := []byte(request.Body)
+	var integration models.Integration
+
+	err := json.Unmarshal(body, &integration)
+	if err != nil {
+		log.Println(err)
+		return events.APIGatewayV2HTTPResponse{
+			StatusCode: 500,
+			Body:       "IntegrationServiceHandler",
+		}, errors.New("error unmarshaling body")
+	}
+
+	// get application data
+	store := store.NewStore()
```

Comment:
@jwagenaar - so here I created a temp store so that I could progress with development, however I would lean more towards an /applications/<id> endpoint that returns an applications data. That way the lambda does not need to do any database calls (only http requests) and that logic is abstracted away.